fuzzy-search.js

```diff
@@ -36,10 +38,10 @@ function fuzzySearch(haystackList, needle, renderer, caseSensitive = false) {
 	};
 
 	// Store preciseMatch and fuzzyMatch separately, to avoid running sort on preciseMatch
-	let preciseMatch = [];
+	let preciseMatch = "";
 	let fuzzyMatch = [];
 
-	for (let i = 0; i < haystackList.length; i++) {
+	for (let i = haystackList.length - 1; i >= 0; i--) {
```

Comment:
Move` haystackList.length ` from for loop's `condition expression` to `initialization expression`, so that the `length` property of the array is called only once